datajunction-server/datajunction_server/api/metrics.py

```diff
@@ -69,17 +71,23 @@ async def list_metrics(
     validate_access: access.ValidateAccessFn = Depends(
         validate_access,
     ),
+    cache: Cache = Depends(get_cache),
+    background_tasks: BackgroundTasks,
```

Comment:
Can this also include a `cache_control` header so that the client can control whether they want the cached value or not? E.g.:
```
Cache-Control: no-cache
Cache-Control: max-age=60
```

Reply:
That makes sense, thanks!